Aztec/Classes/Libxml2/DOM/Attribute.swift

```diff
@@ -5,14 +5,20 @@ extension Libxml2 {
     ///
     class Attribute: CustomReflectable {
         let name: String
-
+        
+        // MARK: - CustomReflectable
+        
+        public var customMirror: Mirror {
+            get {
+                return Mirror(self, children: ["name": name])
+            }
```

Comment:
If you only have a getter, pretty sure you can avoid the `get {}` block. Unless... changed in Swift 3?